tartufo/util.py

```diff
@@ -44,29 +44,42 @@ def convert_regexes_to_rules(regexes: Dict[str, Pattern]) -> Dict[str, Rule]:
     }
 
 
-def echo_issues(
-    issues: "List[Issue]",
-    as_json: bool,
+def echo_result(
+    options: "types.GlobalOptions",
+    scanner: "ScannerBase",
     repo_path: str,
     output_dir: Optional[pathlib.Path],
 ) -> None:
     """Print all found issues out to the console, optionally as JSON.
-
-    :param issues: The list of issues to be printed out
-    :param as_json: Whether the output should be formatted as JSON
+    :param options: Global options object
+    :param scanner: ScannerBase contaning issues and excluded paths from config tree
     :param repo_path: The path to the repository the issues were found in
     :param output_dir: The directory that issue details were written out to
     """
-    if as_json:
+    if options.json:
         output = {
             "project_path": repo_path,
             "output_dir": str(output_dir) if output_dir else None,
-            "found_issues": [issue.as_dict() for issue in issues],
+            "excluded_paths": [str(path.pattern) for path in scanner.excluded_paths],
+            "excluded_signatures": [
+                str(signature) for signature in options.exclude_signatures
+            ],
+            "found_issues": [issue.as_dict() for issue in scanner.issues],
```

Comment:
We should probably add another key in here with the scan time, so we have parity of reporting. Maybe... `scan_time`? Or `scanned_at`? `scan_finish`? `finish_time`? I'm not sure. Naming things is hard.

Reply:
Added `scan_time`